### PR TITLE
Add grunt task to check for correct nls keys

### PR DESCRIPTION
**Pull request by marcelgerber**
This grunt task checks that all nls keys referenced in a nls file are actually present in the root file, too.
This will help us spot typos and the like.

I've dropped all the keys removed from the root file, but still present in some nls files.

### CODE REVIEW

**Comment by abose** [comment]
There are a lot of string deletions in this PR. Is that intended?

**Comment by MarcelGerber** [comment]
Yes, these are intended.
They are all strings that are no longer being used in Brackets.

**Comment by abose** [comment]
Thanks. Could you update the pr resolving merge conflicts?

**Comment by MarcelGerber** [comment]
Ready.

**Comment by abose** [comment]
Merge conflicts again :sweat_smile: probably due to all language pr merges for 1.5.

**Comment by MarcelGerber** [comment]
@abose It's mergeable again.

File: tasks/build.js [hunk 1]
```diff
@@ -246,6 +246,59 @@ module.exports = function (grunt) {
...
+                return rootDefinitions[filename].indexOf(key) < 0;
+            });
+            
+            if (unknownKeys.length) {
+                grunt.log.writeln("There are unknown keys included in " + PATH + "/" + subdir + "/" + filename + ":", unknownKeys);
+                encounteredErrors = true;
+            }
+        });
+
+        if (encounteredErrors) {
```
Comment (ingorichter) [inline tasks/build.js]: Why not call `done(encounteredErrors)`?

**Comment by abose** [comment]
Merged.

**Pull request merged**